src/univariate/continuous/locationscale.jl

```diff
@@ -0,0 +1,73 @@
+doc"""
+    LocationScale(μ,σ,ρ)
+    
+A location-scale transformed distribution with location parameter `μ`,
+scale parameter `σ`, and given distribution `ρ`.
+$f(x) = \frac{1}{σ}ρ(\frac{x-μ}{σ})$
+
+```julia
+LocationScale(μ,σ,ρ) # location-scale transformed distribution
+params(d)            # Get the parameters, i.e. (μ,σ,*****)
+location(d)          # Get the location parameter
+scale(d)             # Get the scale parameter
+```
+
+External links
+[Location-Scale family on Wikipedia](https://en.wikipedia.org/wiki/Location%E2%80%93scale_family)
+"""
+struct LocationScale{T<:Real} <: ContinuousUnivariateDistribution
+    μ::T
+    σ::T
+    ρ::ContinuousUnivariateDistribution
```

Comment:
Should `LocationScale` be parameterized with the subtype of `ContinuousUnivariateDistribution` as well?